spec/acceptance/lesson_email_spec.rb

```diff
@@ -59,6 +59,59 @@
 end
 
 feature %q{
+  As a teacher
+  I want to be sure
+  That notifications contain the correct start time and timezone in calendar.ics
+} do
+
+  background do
+    # Pacific params
+    @school_pacific = create(:school, timezone: "Pacific Time (US & Canada)")
+    @venue_pacific = create(:venue, school: @school_pacific)
+    @admin_pacific = create(:admin, school: @school_pacific)
+    # Eastern params
+    @school_eastern = create(:school, timezone: "Eastern Time (US & Canada)")
+    @venue_eastern = create(:venue, school: @school_eastern)
+    @admin_eastern = create(:admin, school: @school_eastern)
+    @attendee = FactoryGirl.create(:user)
+  end
+
+  scenario "Pacific teacher sending notification email" do
+    expect {
+      create_lesson_and_send_notification(@admin_pacific, @attendee)
+    }.to change{ActionMailer::Base.deliveries.count}.by(1)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.